portlets/src/main/frontend/src/apps/profileStats/components/ProfileStats.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <v-app
+    id="profile-stats-portlet"
+    class="ml-md-2"
+    flat
+    dark>
+    <main>
+      <v-container pa-0>
+        <v-layout
+          row
+          wrap
+          mx-0
+          class="white profileCard">
+          <user-dashboard
+            v-if="!isFlipped"
+            class="profileFlippedCard profileStats"
+            @specificCard="setFlippedCard"/>
+          <v-flex
+            v-if="isFlipped"
+            :is="currentComponent"
+            d-flex
+            xs12
+            sm12
+            class="profileFlippedCard ConnexionsRequests"
+            @isProfileStats="setFlippedCard"/>
+        </v-layout>
+      </v-container>
+    </main>
+  </v-app>
+</template>
+<script>
+    import UserDashboard from './UserDashbord.vue';
+    import ConnectionsRequests from './ConnectionsRequests.vue';
+    import TotalPoints from './TotalPoints.vue';
+    import SpacesRequests from './SpacesRequests.vue';
+    import GamificationRank from "./GamificationRank.vue";
```

Comment:
Declare those in initComponents.js so that you don't need to import it here anymore. This will allow to extend it as well

Reply:
Done!

Reply:
Done!